src/components/Facet.vue

```diff
@@ -50,8 +50,10 @@ export default {
 
       if (facetsToApply && facetsToApply.length) {
         newQuery[facetHeader] = facetsToApply;
+        newQuery['page'] = 1;
       } else {
         delete newQuery[facetHeader];
+        newQuery['page'] = 1;
```

Comment:
Non blocking, just an observation. I think you could just set `newQuery['page'] = 1;` once after the conditional as it gets set to the same thing regardless of the path taken. There is nothing wrong with it like this logically though.